k8s/default_quota.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
I think we can create quota when user register automatically, but the feature can be implemented in another PR.